README.md

```diff
@@ -187,19 +381,21 @@ a new one and any subsequent calls underneath that context manager will be added
 
 If `is_conditional` is `True` and there is no transaction in the call chain, then the connection will not create a new transaction. Instead, the query will be performed without a transaction.
 
-### Creating database tables
+### Creating database and tables
 
-When you decorate a Pydantic model with Pynocular, it creates a SQLAlchemy table as a private variable. This can be accessed via the `_table` property
-(although accessing private variables is not recommended). Using this, along with Pynocular's `create_tracked_table` function, allows you to create tables
-in your database based off of Pydantic models!
+With Pynocular you can use simple python code to create new databases and database tables. All you need is a working connection string to the database host, a DatabaseInfo that contains the information of the database you want to create and a properly decorated pydantic model. When you decorate a Pydantic model with Pynocular, it creates a SQLAlchemy table as a private variable. This can be accessed via the `_table` property
```

Comment:
```suggestion
With Pynocular you can use simple python code to create new databases and database tables. All you need is a working connection string to the database host, a `DatabaseInfo` object that contains the information of the database you want to create, and a properly decorated pydantic model. When you decorate a pydantic model with Pynocular, it creates a SQLAlchemy table as a private variable. This can be accessed via the `_table` property
```